static/templates/actors/character/tabs/crafting.hbs

```diff
@@ -98,10 +98,11 @@
             <div class="action-header">{{localize "PF2E.FormulaKnownTitle"}}</div>
             <ol class="directory-list item-list formula-list">
                 {{!-- Add section for each formula level --}}
-                {{#each crafting.knownFormulas as |section lvl|}}
-                    <li class="formula-level-header formula-header" data-item-type="formulaLevel" data-level="{{lvl}}">
+                {{#each crafting.knownFormulas}}
```

Comment:
For one reason or another we've been leaning off of not using this for handlebars. use as |secetion| and access section.level and section.formulas

Reply:
Done.